### ISSUE TITLE
Move page-up/page-down gestures to opt-in in settings

### ISSUE THREAD

**Issue by bengolds** [connectbot/connectbot#270]
We have virtual keys for pg up and pg down; can we remove the "drag on the left side of the screen" gesture? @kruton @jklein24

**Comment by jklein24** [comment]
:+1: from me on this - I've found the page up/down on the left side of the screen really unexpected before really paying attention to the instructions. It's particularly confusing when not in a context where pgup/dn works correctly. It just sort of makes scrolling seem broken. @kruton, are you ok with removing this? I guess we could always add a setting to keep it there, but I'd rather kill it altogether.

**Comment by khorimoto** [comment]
+1 - I've also found this functionality confusing and unexpected.

**Comment by alescdb** [comment]
Would be nice to make it optional because I'm use to it :-)

**Comment by rhansby** [comment]
Accidentally removed this in #277. Please let me know if we want to keep this feature.

**Comment by jklein24** [comment]
I think we should keep it around in settings because I'm sure @alescdb isn't alone in being used to this behavior.

**Comment by jklein24** [comment]
@rhansby, feel free to do this in another PR once your giant one lands.

**Comment by leonerd** [comment]
+1 for keeping it, even as an option. I verymuch use it all the time - e.g. running IRC clients via screen
